### PR TITLE
Uptake TOTP and recovery code changes and docs

### PR DESCRIPTION
**Pull request by taronish-stytch**

### CODE REVIEW

File: dist/b2b/organizations_members.js [hunk 1]
```diff
@@ -179,6 +179,27 @@ class Members {
...
 
+  /**
+   * @param data {@link B2BOrganizationsMembersTOTPRequest}
+   * @param options {@link B2BOrganizationsMembersTOTPRequestOptions}
+   * @returns {@link B2BOrganizationsMembersTOTPResponse}
+   * @async
+   * @throws A {@link StytchError} on a non-2xx response from the Stytch API
+   * @throws A {@link RequestError} when the Stytch API cannot be reached
+   */
+  totp(data, options) {
```
Comment (ollie-stytch) [inline dist/b2b/organizations_members.js:190]: delete TOTP? Also documentation?

Comment (taronish-stytch) [reply dist/b2b/organizations_members.js:190]: Yeah this one is super strange. even the object names are missing the `DeleteMFA` portion that is present in the protos

**Review by ollie-stytch — APPROVED** [review]
Looks good besides `totp` --> `deleteTotp`

File: dist/b2b/organizations_members.js (hunk 1 above)
Comment (logan-stytch) [reply dist/b2b/organizations_members.js:190]: @taronish-stytch - I can work with you tomorrow to figure out why codegen is doing weird things.

**Pull request merged**